extensions/blocks/transcription/components/speakers-controls.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	DropdownMenu,
+	TextControl,
+	BaseControl,
+	Button,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Fragment, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { formatUppercase } from '../../../shared/icons';
+
+export function AddSpeakerButton( {
+	className,
+	onAdd,
+} ) {
+	const [ value, setValue ] = useState();
+
+	function addNewSpeakerHandler() {
+		onAdd( value );
+		setValue( '' );
+	}
+
+	return (
+		<BaseControl>
+			<div className={ `${ className }__speaker` }>
+				<TextControl
+					label={ __( 'Add a speaker', 'jetpack' ) }
+					value={ value }
+					onChange={ setValue }
+					onKeyDown={ ( { key } ) => {
+						if ( key !== 'Enter' ) {
+							return;
+						}
+
+						addNewSpeakerHandler();
```

Comment:
Would it be worth making this positive? 
```suggestion
						key === 'Enter' && addNewSpeakerHandler();
```